model-optimizer/extensions/front/softplus.py

```diff
@@ -0,0 +1,39 @@
+"""
+ Copyright (C) 2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from extensions.ops.activation_ops import Exp, Log
+from extensions.ops.elementwise import Add
+from mo.front.common.partial_infer.utils import float_array
+from mo.front.common.replacement import FrontReplacementOp
+from mo.front.tf.graph_utils import create_op_node_with_second_input
+from mo.graph.graph import Graph
+
+
+class SoftPlus(FrontReplacementOp):
+    op = 'SoftPlus'
+    enabled = True
+
+    def replace_sub_graph(self, graph: Graph, match: dict):
+        softplus = match['op']
+
+        exp_node = Exp(graph, {'name': softplus.name + '/Exp'}).create_node()
```

Comment:
It is not safe to write `softplus.name`. Instead write `name = softplus.soft_get('name', softplus.id)` and use variable `name` everywhere.